engine/hid/src/native/hid_gamepad_driver_glfw.cpp

```diff
@@ -140,7 +138,12 @@ namespace dmHID
 
     static void GLFWGamepadCallback(int gamepad_id, int connected)
     {
-        GLFWEnsureAllocatedGamepad(g_GLFWGamepadDriver, gamepad_id);

```

Comment:
We only ever set the gamepad connection status to true and we ignore the `connected` parameter. This must be a regression from when we refactored the HID code and added a concept of drivers and added support for DInput.